src/main.js

```diff
@@ -46,6 +46,14 @@ import {
 } from '@fortawesome/free-brands-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
 import ga from './ga';
+import electron from './electron';
+// Find out if we are are running inside electon
+if (
+  typeof navigator === 'object' &&
```

Comment:
I prefer to use the lodash helpers, they catch more corner cases. import isObject and isString/

https://lodash.com/docs/4.17.15#isObject
https://lodash.com/docs/4.17.15#isString

Use `includes` as it's convey's intent better than `indexOf`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes